server/src/main/kotlin/suwayomi/tachidesk/graphql/dataLoaders/ChapterDataLoader.kt

```diff
@@ -119,6 +119,27 @@ class BookmarkedChapterCountForMangaDataLoader : KotlinDataLoader<Int, Int> {
         }
 }
 
+class HasDuplicateChaptersForMangaDataLoader : KotlinDataLoader<Int, Boolean> {
+    override val dataLoaderName = "HasDuplicateChaptersForMangaDataLoader"
+
+    override fun getDataLoader(): DataLoader<Int, Boolean> =
+        DataLoaderFactory.newDataLoader { ids ->
+            future {
+                transaction {
+                    addLogger(Slf4jSqlDebugLogger)
+                    val duplicatedChapterCountByMangaId =
+                        ChapterTable.slice(ChapterTable.manga, ChapterTable.chapter_number, ChapterTable.chapter_number.count())
+                            .select { ChapterTable.manga inList ids }
+                            .groupBy(ChapterTable.manga, ChapterTable.chapter_number)
```

Comment:
Need to filter out chapters with a negative chapter number

Reply:
thats possible?

Reply:
-1 is the default value if we cannot get a chapter number